blog/2021-04-09-astral-2.2.md

```diff
@@ -0,0 +1,36 @@
+---
+author: Martin Hjelmare
+authorURL: https://github.com/MartinHjelmare
+authorTwitter: MartinHjelmare
+title: Astral upgraded to version 2.2
+---
+
+We recently merged a [pull request](https://github.com/home-assistant/core/pull/48573) to upgrade the astral library version used in Home Assistant Core to version 2.2. This is a major version bump which includes some breaking changes. We have updated the built in helpers and integrations that depend on astral. This resulted in a couple of breaking changes to our sun helpers.
```

Comment:
```suggestion
We recently merged a [pull request](https://github.com/home-assistant/core/pull/48573) to upgrade the astral library version used in Home Assistant Core to version 2.2. This will be released with Home Assistant 2021.5. This is a major version bump of Astral which includes some breaking changes, this caused us to update our built-in helpers and integrations that depend on astral. This has resulted in a couple of breaking changes to our sun helpers.
```